subsys/bluetooth/audio/bap_endpoint.h

```diff
@@ -73,6 +73,11 @@ struct bt_audio_broadcast_stream_data {
 	/** Codec Specific Data */
 	uint8_t data[CONFIG_BT_AUDIO_CODEC_CFG_MAX_DATA_SIZE];
 };
+
+struct bt_bap_broadcast_source_bis {
+	/** Bis specific codec config */
+	struct bt_audio_codec_cfg codec_cfg;
+};
```

Comment:
Given that this struct only contains a single field, we could probably omit the struct and just use `struct bt_audio_codec_cfg` in the below array, right?